README.md

```diff
@@ -11,9 +9,7 @@
 - Frameworks
 - Technologies used
 
-## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
 
 
 ## Getting Started
```

Comment:
- [ ] I think it would be better to customize `Getting Started` section to explain to others about how to download and use your project. 👌

> For Example 💡
To get a local copy run `git clone git@github.com:SabaAhmad786/hello.git`
Open `index.html` on any browser